src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -7560,6 +7560,78 @@ def test_aks_create_with_enable_network_observability(self, resource_group, reso
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2',
+                                    preserve_default_location=True)
+    def test_aks_create_with_enable_cost_analysis(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=78890009&view=results) to validate the change.

Reply:
Requeued [new one](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=78954450&view=results)